index.js

```diff
@@ -57,14 +57,14 @@ module.exports = function (input) {
   if (!validAmount(amount) || !parsableUnit(unit)) {
     throw 'Can\'t interpret ' + (input || 'a blank string');
   }
-  if (unit === '') return Number(amount);
+  if (unit === '') return Math.ceil(Number(amount));
```

Comment:
should probably be `Math.round`, rather than `ceil()`. `1.0000000000000001` would parse to `2` with this